foruma-frontend/src/stories/Button.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <button type="button" :class="classes" @click="onClick" :style="style">{{ label }}</button>
```

Comment:
Codacy has a fix for the issue: [Replace `{{·label·}}` with `⏎····{{·label·}}⏎··`](https://app.codacy.com/gh/agabani/foruma/pullRequest?prid=7388752)
```suggestion
  <button type="button" :class="classes" @click="onClick" :style="style">
```